.github/workflows/docker.yaml

```diff
@@ -0,0 +1,29 @@
+name: Build and Publish docker image to ghcr
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
To build multi-arch I think we need another step to install the `qemu` dependency. [Example](https://github.com/redhat-actions/buildah-build/blob/main/.github/workflows/multiarch.yml#L38) here.

Reply:
That's another issue :D

The previous one was because `buildah` which is a builder from RedHat has naming of `arm` for `arm/v7` arch and it caused conflict.